apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/class-full-site-editing.php

```diff
@@ -61,6 +61,7 @@ private function __construct() {
 		add_action( 'pre_delete_term', array( $this, 'restrict_template_taxonomy_deletion' ), 10, 2 );
 		add_action( 'transition_post_status', array( $this, 'restrict_template_drafting' ), 10, 3 );
 		add_action( 'admin_menu', array( $this, 'remove_wp_admin_menu_items' ) );
+		add_filter( 'block_editor_rest_api_preload_paths', array( $this, 'preload_template_parts' ), 10, 2 );
```

Comment:
This filter isn't available until WP 5.8, but preloading isn't required for editing template parts to work.